RecoTauTag/RecoTau/plugins/RecoTauPiZeroStripPlugin3.cc

```diff
@@ -0,0 +1,393 @@
+      function->SetParameter(idxParameter, parameter);
+    }
+    return function;
+  }
+}
+
+RecoTauPiZeroStripPlugin3::RecoTauPiZeroStripPlugin3(const edm::ParameterSet& pset, edm::ConsumesCollector &&iC)
+  : RecoTauPiZeroBuilderPlugin(pset, std::move(iC)),
+    vertexAssociator_(pset.getParameter<edm::ParameterSet>("qualityCuts"), std::move(iC)), qcuts_(nullptr), etaAssociationDistance_(nullptr), phiAssociationDistance_(nullptr)
+{
+  minGammaEtStripSeed_ = pset.getParameter<double>("minGammaEtStripSeed");
+  minGammaEtStripAdd_ = pset.getParameter<double>("minGammaEtStripAdd");
+
+  minStripEt_ = pset.getParameter<double>("minStripEt");
+  
+  edm::ParameterSet qcuts_pset = pset.getParameterSet("qualityCuts").getParameterSet("signalQualityCuts");
+//-------------------------------------------------------------------------------
+// CV: disable track quality cuts for PFElectronsPFElectron
+//       (treat PFElectrons like PFGammas for the purpose of building eta-phi strips)
+  applyElecTrackQcuts_ = pset.getParameter<bool>("applyElecTrackQcuts");
+  if ( !applyElecTrackQcuts_ ) {
+    qcuts_pset.addParameter<double>("minTrackPt", std::min(minGammaEtStripSeed_, minGammaEtStripAdd_));
+    qcuts_pset.addParameter<double>("maxTrackChi2", 1.e+9);
+    qcuts_pset.addParameter<double>("maxTransverseImpactParameter", 1.e+9);
+    qcuts_pset.addParameter<double>("maxDeltaZ", 1.e+9);
+    qcuts_pset.addParameter<double>("minTrackVertexWeight", -1.);
+    qcuts_pset.addParameter<unsigned>("minTrackPixelHits", 0);
+    qcuts_pset.addParameter<unsigned>("minTrackHits", 0);
+  }
+//-------------------------------------------------------------------------------
+  qcuts_pset.addParameter<double>("minGammaEt", std::min(minGammaEtStripSeed_, minGammaEtStripAdd_));
+  //qcuts_ = new RecoTauQualityCuts(qcuts_pset);
+  std::unique_ptr<RecoTauQualityCuts> qcuts_(new RecoTauQualityCuts(qcuts_pset));
+
+  inputPdgIds_ = pset.getParameter<std::vector<int> >("stripCandidatesParticleIds");
+  edm::ParameterSet stripSize_eta_pset = pset.getParameterSet("stripEtaAssociationDistance");
+  etaAssociationDistance_ = makeFunction("etaAssociationDistance", stripSize_eta_pset);
+  edm::ParameterSet stripSize_phi_pset = pset.getParameterSet("stripPhiAssociationDistance");
+  phiAssociationDistance_ = makeFunction("phiAssociationDistance", stripSize_phi_pset);
+
+  updateStripAfterEachDaughter_ = pset.getParameter<bool>("updateStripAfterEachDaughter");
+  maxStripBuildIterations_ = pset.getParameter<int>("maxStripBuildIterations");
+
+  combineStrips_ = pset.getParameter<bool>("makeCombinatoricStrips");
+  if ( combineStrips_ ) {
+    maxStrips_ = pset.getParameter<int>("maxInputStrips");
+    combinatoricStripMassHypo_ = pset.getParameter<double>("stripMassWhenCombining");
+  }
+
+  verbosity_ = ( pset.exists("verbosity") ) ?
+    pset.getParameter<int>("verbosity") : 0;
+}
+  
+RecoTauPiZeroStripPlugin3::~RecoTauPiZeroStripPlugin3()
```

Comment:
remove this method (from .cc and .h); ... only keep if it stops compiling (if it's somehow required  .. I'm not checking this carefully now)